src/main/java/org/somerville/swag/data/entity/Order.java

```diff
@@ -60,6 +60,22 @@ public BigDecimal getTotal() {
 
     public String getFormattedTotal() { return "£" + getTotal().setScale(2, RoundingMode.HALF_UP); }
 
+    public String getReceipt(String forename) {
+        StringBuilder receipt = new StringBuilder("Thanks For Ordering " + forename + "! Your Swag Will Be With You Soon :D\n\nPurchased Swag:");
+        for (OrderLine orderLine : orderLines) {
+            Product product = orderLine.getProduct();
+            receipt.append("\n")
```

Comment:
Great String building

Reply:
Thank you! I opted for StringBuilder as it's more efficient and doesn't waste memory.